packages/indexer/src/tests/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./setup";
```

Comment:
Rename the folder to `testing`, then export it from `package.json` as `./testing`. That way users will be able to import the functions as `@apibara/indexer/testing`.